### ISSUE TITLE
The code's learning rate hyperparameters seems inconsistent with the paper.

### ISSUE THREAD

**Issue by GooLiang** [allenhaozhu/SSGC#18]
Hello, the learning rate of cora, citeseer, and pubmed is 0.02 in the paper. But it seems to be 0.2 in the implementation. Which one is correct?
In addition, can you provide the code of reddit, or the learning rate of reddit.

**Comment by allenhaozhu** [comment]
It is! I'm sorry for not getting back to you sooner. I think you are right.
As I recall, just keep the same hyperparameter as the SGC. You can have a
try. Basically, the hyperparameters of SGC and SSGC are very similar.

liangyx ***@***.***> 于2023年9月9日周六 00:35写道：

**Comment by GooLiang** [comment]
Thank you for your reply.
On the arxiv code, you provided two versions MLP.py and SSGC-MLP.py. From the implementation point of view, is MLP.py the actual SSGC+MLP of the paper?
BTW, can you provide the reproducible hyperparameters of SSGC+MLP on the arxiv dataset？

**Comment by allenhaozhu** [comment]
Sorry, I have moved from Canberra to Sydney, and the record was missing. I
remember that I kept the graph convolution result first, then trained on it
with MLP. I did not use the specific parameter for the original raw
features. Thus, it is an average pooling for different hops features. Then,
I only tried different hyper-parameters on MLP.

liangyx ***@***.***> 于2023年10月4日周三 03:18写道：
